### PR TITLE
fix: fixes eslint and prettier errors

### PR DESCRIPTION
**Pull request by ChristianMLux**
fixes code breaking errors so i can start to work

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -10,6 +10,12 @@ module.exports = {
   rules: {
     "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+    "prettier/prettier": [
+      "error",
+      {
+        endOfLine: "auto",
+      },
+    ],
```
Comment (ChristianMLux) [inline .eslintrc.js:18]: needed to get rid of prettier/eslint errors. eslint doesn't like any chars at the end of a line. 
we as team wanted to have chars like , and ; at the end of the line, so this fix ist needed to stick to the team rules

File: src/main.js [hunk 2]
```diff
@@ -3,7 +3,6 @@ import App from "./App.vue";
 import router from "./router";
 import MainButton from "./components/MainButton/MainButton.vue";
 
-
```
Comment (ChristianMLux) [inline src/main.js:6]: just got rid of a empty line

File: src/views/Home.vue [hunk 3]
```diff
@@ -9,9 +9,9 @@ export default {
   name: "Home",
   components: {
     LandingPage,
-},
- mounted() {
+  },
+  mounted() {
```
Comment (ChristianMLux) [inline src/views/Home.vue:13]: wrong tabs

File: src/components/QuestionList/QuestionList.vue [hunk 4]
```diff
@@ -248,7 +248,7 @@ ol > li {
 .filter-options {
   display: flex;
   flex-flow: row;
-  align-items: start;
+  align-items: flex-start;
```
Comment (ChristianMLux) [inline src/components/QuestionList/QuestionList.vue:251]: start is not as supported as flex-start

**Review by robinboehm — APPROVED** [review]
LGTM

**Pull request merged**